src/sockperf.cpp

```diff
@@ -2310,7 +2311,7 @@ void cleanup() {
                 if (g_fds_array[ifd]->is_multicast) {
                     FREE(g_fds_array[ifd]->memberships_addr);
                 }
-                FREE(g_fds_array[ifd]);
+                delete g_fds_array[ifd];
```

Comment:
Are you going to replace all MALLOC/FREE ? What is a purpose to do this change just in this place

Reply:
No, I replaced allocator only for `fds_data` structures to properly initialize them.